packages/common/src/types/warehouse.ts

```diff
@@ -15,10 +15,19 @@ export type WarehouseCatalog = {
     };
 };
 
+export type WarehouseTablesCatalog = {
```

Comment:
The same WarehouseCatalog was used for both fields and tables. 

Tables was returning an empty list of fields, but this was uncompatible with the new `partitionColumn` property , so I created a new type which is used only metadata on tables